.github/workflows/reusable-build.yml

```diff
@@ -96,15 +65,15 @@ jobs:
           TIMESTAMP="$(date +%Y%m%d)"
           VARIANT="${{ matrix.fedora_version }}"
 
-          if [[ "${{ matrix.fedora_version }}" -eq "38" ]]; then
+          if [[ "${{ matrix.fedora_version }}" -eq "39" ]]; then
             IS_LATEST_VERSION=false
             IS_STABLE_VERSION=true
             IS_GTS_VERSION=true
-          elif [[ "${{ matrix.fedora_version }}" -eq "39" ]]; then
+          elif [[ "${{ matrix.fedora_version }}" -eq "40" ]]; then
             IS_LATEST_VERSION=true
             IS_STABLE_VERSION=true
             IS_GTS_VERSION=false
-          elif [[ "${{ matrix.fedora_version }}" -eq "40" ]]; then
+          elif [[ "${{ matrix.fedora_version }}" -eq "41" ]]; then
```

Comment:
I know we don't really need this, but no point ripping this out when it'll be added in 4 months again.